src/components/fields/wrapper/wrapper.tsx

```diff
@@ -0,0 +1,77 @@
+import isEmpty from "lodash/isEmpty";
+import React, { useEffect, useState } from "react";
+import { Controller, useFormContext } from "react-hook-form";
+import { EFieldType, IGenericFieldProps, TFrontendEngineFieldSchema } from "src/components/frontend-engine/types";
+import * as FrontendEngineFields from "../";
+import { IWrapperSchema } from "./types";
+
+interface IWrapperProps {
+	id?: string;
+	schema?: IWrapperSchema | undefined;
+	/** only used internally by FrontendEngine */
+	children?: Record<string, TFrontendEngineFieldSchema>;
+}
+
```

Comment:
note: wrapper component is used via 2 scenarios
- from JSON schema: use `schema` prop
- from `FrontendEngine`: use `children` prop